### PR TITLE
docs: Improve docs in plugin-jdbc and plugin-jdbc-arrow-flight

### PR DESCRIPTION
**Pull request by shrutimantri**
Improved docs for plugin-jdbc and plugin-jdbc-arrow-flight

### CODE REVIEW

File: plugin-jdbc-arrow-flight/src/main/java/io/kestra/plugin/jdbc/arrowflight/Query.java [hunk 1]
```diff
@@ -29,21 +29,25 @@
 @Plugin(
     examples = {
         @Example(
-            title = "Send a SQL query to a database and fetch a row as outputs using Apache Arrow Flight SQL driver",
+            title = "Send a SQL query to a database and fetch row(s) using Apache Arrow Flight SQL driver",
             code = {
+                "id: arrow-flight-sql-query",
+                "type: io.kestra.plugin.jdbc.arrowflight.Query",
```
Comment (loicmathieu) [inline plugin-jdbc-arrow-flight/src/main/java/io/kestra/plugin/jdbc/arrowflight/Query.java:35]: This is automatically added via our documentation generation tool for non-full examples.
Please remove them

File: plugin-jdbc-arrow-flight/src/main/java/io/kestra/plugin/jdbc/arrowflight/Query.java [hunk 2]
```diff
@@ -29,21 +29,25 @@
...
                 "sql: select * FROM departments",
                 "fetch: true",
             }
         ),
         @Example(
-            title = "Send a sql query to a Dremio coordinator and fetch a row as outputs using Apache Arrow Flight SQL driver",
+            title = "Send a sql query to a Dremio coordinator and fetch rows as outputs using Apache Arrow Flight SQL driver",
             code = {
+                "id: arrow-flight-sql-query",
+                "type: io.kestra.plugin.jdbc.arrowflight.Query",
```
Comment (loicmathieu) [inline plugin-jdbc-arrow-flight/src/main/java/io/kestra/plugin/jdbc/arrowflight/Query.java:47]: Same here, remove them

File: plugin-jdbc/src/main/java/io/kestra/plugin/jdbc/AbstractJdbcQuery.java [hunk 3]
```diff
@@ -193,26 +193,26 @@ private Object convertCell(int columnIndex, ResultSet rs, AbstractCellConverter
...
         @Schema(
-            title = "Lit of map containing rows of fetched data",
+            title = "List of map containing rows of fetched data.",
             description = "Only populated if 'fetch' parameter is set to true."
         )
         private final List<Map<String, Object>> rows;
 
         @Schema(
-            title = "The url of the result file on kestra storage (.ion file / Amazon Ion text format)",
+            title = "The uri of the result file on Kestra storage (.ion file / Amazon Ion formatted text file).",
```
Comment (loicmathieu) [inline plugin-jdbc/src/main/java/io/kestra/plugin/jdbc/AbstractJdbcQuery.java:209]:
```suggestion
            title = "The URI of the result file on Kestra's internal storage (.ion file / Amazon Ion formatted text file).",
```

File: plugin-jdbc/src/main/java/io/kestra/plugin/jdbc/AutoCommitInterface.java [hunk 4]
```diff
@@ -5,12 +5,12 @@
 
 public interface AutoCommitInterface {
     @Schema(
-        title = "If autocommit is enabled",
+        title = "This is true if autocommit is enabled, else false.",
```
Comment (loicmathieu) [inline plugin-jdbc/src/main/java/io/kestra/plugin/jdbc/AutoCommitInterface.java:8]:
```suggestion
        title = "Whether autocommit is enabled.",
```

**Review by anna-geller — APPROVED** [review]
LGTM, ty!

File: plugin-jdbc-arrow-flight/src/main/java/io/kestra/plugin/jdbc/arrowflight/Query.java (hunk 1 above)
Comment (shrutimantri) [reply plugin-jdbc-arrow-flight/src/main/java/io/kestra/plugin/jdbc/arrowflight/Query.java:35]: Ohh, I see. I will remove this.

**Review by loicmathieu — APPROVED** [review]
LGTM, thanks a lot.

**Pull request merged**